examples/js_of_ocaml/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Js_of_ocaml • TodoMVC</title>
```

Comment:
I think this is the official spelling.

On Fri, Jun 26, 2015, 22:48 Arthur Verschaeve <notifications@github.com>
wrote:

> In examples/js_of_ocaml/index.html
> <https://github.com/tastejs/todomvc/pull/1339#discussion_r33399033>:
>
> > @@ -0,0 +1,13 @@
> > +<!doctype html>
> > +<html lang="en">
> > +  <head>
> > +    <meta charset="utf-8">
> > +    <meta name="viewport" content="width=device-width, initial-scale=1">
> > +    <title>Js_of_ocaml • TodoMVC</title>
>
> Let's properly write this instead of snake_case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tastejs/todomvc/pull/1339/files#r33399033>.
>